apps/platform/src/components/ui/combobox.tsx

```diff
@@ -37,19 +38,40 @@ import {
   DialogTrigger
 } from './dialog'
 import { Button } from './button'
+import { WorkspaceSchema } from '@keyshade/schema/schemas'
+
+interface WorkspaceResponse {
```

Comment:
The response object already have the pagination type included. I would highly recommend you to go through the entire schema package once to understand the types.